Sources/OutlineView/AdjustableSeparatorRowView.swift

```diff
@@ -2,7 +2,8 @@ import AppKit
 import ObjectiveC
 
 /// An NSTableRowView with an adjustable separator line.
-class AdjustableSeparatorRowView: NSTableRowView {
+@available(macOS 11.0, *)
```

Comment:
This is no needed here.

Reply:
This is not strictly required, but I added it to enforce the macOS requirement at compile time, as `AdjustableSeparatorRowView` can only be used on macOS 11 and up for now.